src/resources/views/auth/login.blade.php

```diff
@@ -25,5 +26,12 @@
 
   </div>
   <!-- /.login-box-body -->
+  @else
```

Comment:
How should we login if we already have an account ?

Reply:
https://github.com/herpaderpaldent/web/blob/a34205db19eaf93f615a322666d648b70d6e860d/src/Http/Controllers/Auth/SsoController.php#L154-L166

This allows to login into existing accounts. However the error is not presented to the user. I don't know why exactly but i suspect it has to do with the `send()` method. 
However this is needed as if i'd be using return the controller claims that it expects a return statement of the type User. 

Here i am very open for better solutions.

Reply:
There is another option:
If the registration is disbled show such an information on the auth/login page everytime someone visiting stating f.e. "Registration of new characters has been disabled, however existing users will be able to login just as they are used to".

Reply:
Your notification is not displayed because you're not returning it but only exiting the function.
This have to be done in the called method : https://github.com/eveseat/web/blob/ab10fb4c0c8e83c62003ff4d02fab678244f0ec9/src/Http/Controllers/Auth/SsoController.php#L63

You can maybe handle this by checking if both settings is false and `$user` is null